core/testing.go

```diff
@@ -27,21 +20,24 @@ func MockConfig(t *testing.T) *Config {
 }
 
 // StartMockNode starts a mock Core node background process and returns it.
-func StartMockNode() *node.Node {
+func StartMockNode(suppressStdout bool) *node.Node {
```

Comment:
Passing in configuration parameters as individual function arguments is a bit...messy. Not necessary to change, but it would be nice if there was a cleaner way of doing this, e.g. a configuration object?